Resources/translations/time.uk.xliff

```diff
@@ -4,27 +4,27 @@
         <body>
             <trans-unit id="1">
                 <source>diff.ago.year</source>
-                <target>1 рiк тому|%count% роки тому|%count% рокiв тому</target>
+                <target>%count% рiк тому|%count% роки тому|%count% рокiв тому</target>
```

Comment:
@pilot you can check [russian locale](https://github.com/KnpLabs/KnpTimeBundle/blob/master/Resources/translations/time.ru.xliff#L7) i.e.